src/FakeItEasy.nuspec

```diff
@@ -18,16 +18,6 @@
       <group targetFramework=".NETStandard1.6">
         <dependency id="Microsoft.Extensions.DependencyModel" version="1.0.0" />
         <dependency id="System.Collections.Concurrent" version="4.0.12" />
-        <dependency id="System.Console" version="4.0.0" />
-        <dependency id="System.Diagnostics.Tools" version="4.0.1" />
-        <dependency id="System.Diagnostics.Debug" version="4.0.11" />
-        <dependency id="System.IO.FileSystem" version="4.0.1" />
-        <dependency id="System.IO.FileSystem.Primitives" version="4.0.1" />
-        <dependency id="System.Linq.Expressions" version="4.1.0" />
-        <dependency id="System.Reflection" version="4.1.0" />
-        <dependency id="System.Reflection.Emit" version="4.0.1" />
-        <dependency id="System.Reflection.Extensions" version="4.0.1" />
-        <dependency id="System.Reflection.TypeExtensions" version="4.1.0" />

```

Comment:
This worries me a little... have you checked that the generated nupkg works as expected?